packages/core/src/App/Containers/Modals/VerificationDocumentSubmitted/__test__/verification-document-submitted.spec.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import VerificationDocumentSubmitted from '../verification-document-submitted';
+
+describe('<VerificationDocumentSubmited />', () => {
+    let modal_root_el: HTMLDivElement;
+
+    beforeAll(() => {
+        modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+
+    it('should not render the VerificationDocumentSubmited component', async () => {
+        const mock = mockStore({
+            ui: {
+                is_verification_submitted: false,
+            },
+        });
+
+        const wrapper = ({ children }: { children: JSX.Element }) => (
+            <StoreProvider store={mock}>{children}</StoreProvider>
+        );
+        render(<VerificationDocumentSubmitted />, { wrapper });
+        expect(screen.queryByTestId('dt_div_100_vh')).not.toBeInTheDocument();
```

Comment:
can we add
expect(
            screen.getByText(
                'We’ll need 1 - 3 days to review your documents and notify you by email. You can practice with demo accounts in the meantime.'
            )
        ).not.toBeInTheDocument();